### PR TITLE
[stable/rabbitmq] Add additional liveness/readiness probe parameters

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Severn Tsui <stsui@carnegietechnologies.com>

**What this PR does / why we need it**: Adds the ability to configure probing period and thresholds for both liveness and readiness checks.

@bitnami-bot

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
Please also add in the README.md

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 2.1.3
+version: 2.1.4
```
Comment (juan131) [inline stable/rabbitmq/Chart.yaml]: Increase minor version.

**Review by juan131 — CHANGES_REQUESTED** [review]
Please also update the **values-production.yaml** file with the same changes you introduced on **values.yaml**

**Review by cpanato — APPROVED** [review]

**Pull request merged**